js/timepicker.js

```diff
@@ -116,14 +116,16 @@
       this._handleClockClickStartBound = this._handleClockClickStart.bind(this);
       this._handleDocumentClickMoveBound = this._handleDocumentClickMove.bind(this);
       this._handleDocumentClickEndBound = this._handleDocumentClickEnd.bind(this);
+      this._inputFromTextFieldBound = this._handleTimeInputEnterKey.bind(this);
 
```

Comment:
User experience advice : the clock should be updated when typing (keyup) or on input focus lost, instead of pressing the enter key.

I will not block the merge due to this comment, as this is not the scope of the issue yet.

Reply:
I think I get what you mean. I will just update `keydown` to `keyup`.